src/View/SSViewer_Scope.php

```diff
@@ -359,27 +343,90 @@ public function next()
             return false;
         }
 
-        return $this->itemIterator->key();
+        return true;
```

Comment:
The key was never used, so it's easier to just have a straight boolean. This is used to determine if the while loop should continue or not in the parser.